package.json

```diff
@@ -2,7 +2,7 @@
   "name": "animated",
   "version": "0.2.0",
   "description": "Declarative Animations Library for React and React Native",
-  "main": "lib/index.js",
+  "main": "lib/targets/react-dom",
```

Comment:
Ah sorry. The `index.js` doesn't exist anymore.
Now there is just `AnimatedImplementation` and 2 targets.

Do you have an idea on how we can swap `main` entry that when running on RN it points to `lib/targets/react-native` and on web `lib/targets/react-native`.

I know the `react-native` packager will look at `react-native` key in package.json. However not sure how that works inside of a library.

All in all. Do you know of a way I can support you on this? Add back index.js that just exports the RN implementation? Then also exports `web`?